tests/ChannelTest.php

```diff
@@ -13,6 +13,13 @@
 
 class ChannelTest extends TestCase
 {
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->withoutDeprecationHandling();
```

Comment:
This configures Laravel to let deprecation notices bubble up to the PHPUnit error handler.